app/controllers/partners/invitations_controller.rb

```diff
@@ -0,0 +1,50 @@
+# This exists so that we can override some of the devise resource
+class Partners::InvitationsController < Devise::InvitationsController
+  layout "devise"
+
+  skip_before_action :authorize_user
+  skip_before_action :authenticate_user!
+  # This one causes a redirect require_no_authentication
+  skip_before_action :require_no_authentication
+
+  def update
+    raw_invitation_token = update_resource_params[:invitation_token]
+    self.resource = accept_resource
+    invitation_accepted = resource.errors.empty?
+
+    yield resource if block_given?
+
+    if invitation_accepted
+      if redirect_to_legacy_partnerbase_app?
+        redirect_to legacy_partnerbase_login_url
```

Comment:
I was really scared by this whole controller but @scooter-dangle and I realized you copied update from https://github.com/scambra/devise_invitable/blob/f6a308d427de1bd263337be4a334f586962ce589/app/controllers/devise/invitations_controller.rb#L45-L68 and added these few lines for the redirect? This means we can get rid of this file after onebase is finished right? Can we leave a comment here so people know where to find the original?

Reply:
👍 yep I'll add comments here :)